src/components/dropdown-button/dropdown-button.stories.tsx

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import { Story, Meta } from '@storybook/react/types-6-0';
+import { Box } from 'rebass';
+import { action } from '@storybook/addon-actions';
+
+import DropdownButton, { DropdownButtonProps } from './index';
+import { IconName } from '../icon';
+import IconButton from '../icon-button';
+
+export default {
+  title: 'Quartz/DropdownButton',
+  component: DropdownButton,
+} as Meta;
+
+const argTypes = {
+  alignLeft: {
+    control: {
+      type: 'boolean',
+    },
+    type: {
+      required: false,
+      summary: 'choose to open dropdown on left',
+    },
+    defaultValue: false,
+  },
```

Comment:
hey, @ehsan-github 
I think this type of declaring `argTypes` is a tad outdated.
Storybook doesn't like when the summary is in `type>summary`
in general, in the latest PRs I've been moving the descriptions inside the `Props` as doc comments

can you please have a look at this PR https://github.com/logicalclocks/quartz/pull/270?
I think this way the stories are more concise
and keeping the descriptions for the props in the Props helps with code readability

also, I've been moving things to a convention you can see in the `user` component

```
styles.ts
User.ts
stories.ts
index.ts // returns `User` and all of the props
```

wdyt?

Reply:
> hey, @ehsan-github I think this type of declaring `argTypes` is a tad outdated. Storybook doesn't like when the summary is in `type>summary` in general, in the latest PRs I've been moving the descriptions inside the `Props` as doc comments
> 
> can you please have a look at this PR #270? I think this way the stories are more concise and keeping the descriptions for the props in the Props helps with code readability
> 
> also, I've been moving things to a convention you can see in the `user` component
> 
> ```
> styles.ts
> User.ts
> stories.ts
> index.ts // returns `User` and all of the props
> ```
> 
> wdyt?

nice job with enabling the storybooks docs addon! It makes writing stories easier and will make component code more readable since the documentation for each property is just near it.